app/assets/javascripts/application.js

```diff
@@ -2,6 +2,7 @@
 //= require turbolinks
 //= require i18n
 //= require jquery.ui.autocomplete
+//= require jquery.ui.spinner
```

Comment:
can you provide more info on why patternfly spinner is not enough? I was hoping to get rid of jquery.ui when we rewrite the search autocompleter.... /cc @Rohoover